alf/environments/parallel_environment.cpp

```diff
@@ -44,31 +44,25 @@ namespace py = pybind11;
 
 const size_t addr_alignment = 16;
 
+// The size of job queue.
+// Since we always wait for job to finish before sending the next job,
+// we only need a shoft job queue.
+const size_t kJobQueueSize = 3;
+
 inline size_t align(size_t x) {
   return ((x + addr_alignment - 1) / addr_alignment) * addr_alignment;
 }
 
 // TODO(emailweixu): directly use cnest c++ function
```

Comment:
remove TODO?

Reply:
I think Wei means using the one directly from c++, not via python binding.